doc/sr_integration.rst

```diff
@@ -0,0 +1,211 @@
+*****************************
+Integrating into a Simulation
+*****************************
+
+========
+Overview
+========
+
+This document provides some general guidelines to integrate the SmartRedis client
+into existing simulation codebase. Developers of these simulation codebases will
+need to identify the exact places to add the code, generally only SmartRedis
+calls will need to be added in two places:
+
+1. Initialization
+2. Main loop
+
+==============
+Initialization
+==============
+
++++++++++++++++++++
+Creating the client
++++++++++++++++++++
+
+The SmartRedis client must be initialized before it can be used to communicate
+with the database. In the C++ and Python versions of the clients, this is done
+when creating a new client whereas in the C and Fortran client an `initialize`
+method that must be called.
+
```

Comment:
Change to:  " In the C++ and Python versions of the clients, this is done
when creating a new client, whereas in the C and Fortran client an `initialize`
method must be called."

Reply:
Nice catch.